drivers/hmis/app/graphql/types/base_object.rb

```diff
@@ -123,7 +125,7 @@ def activity_log_object_identity
 
         raise "Missing #{self.class.graphql_name}::object[:id] in #{object.inspect}" unless object.key?(:id)
 
-        object.fetch[:id]
+        object.fetch(:id)
```

Comment:
Flagging this for sanity checking/discussion. After making the `CeReferralSwimlane` schema object backed by a hash instead of an ActiveRecord, I started to see the previous code throwing errors like: `wrong number of arguments (given 0, expected 1..2)`. (To reproduce: revert this change and run `rspec drivers/hmis/spec/requests/hmis/ce/ce_referral_spec.rb`)

Reply:
Our convention so far has been to use `OpenStruct` to back gql types when there is no model to back it. There are several examples if you grep openstruct inside the graphql directory. I would suggest switching to that